packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -141,29 +172,51 @@ export default function LabeledField(props: Props) {
     }
 
     function maybeRenderError(): React.ReactNode | null | undefined {
-        if (!error) {
-            return null;
-        }
-
         return (
             <React.Fragment>
                 <Strut size={spacing.small_12} />
-                <LabelSmall
-                    style={light ? styles.lightError : styles.error}
-                    role="alert"
-                    id={errorId}
-                    testId={testId && `${testId}-error`}
+                <View
+                    style={styles.errorSection}
+                    id={errorSectionId}
+                    testId={testId && `${testId}-error-section`}
+                    aria-live="assertive"
                 >
-                    {error}
-                </LabelSmall>
+                    {error && (
+                        <>
+                            <PhosphorIcon
+                                icon={WarningCircle}
+                                style={[
+                                    styles.errorIcon,
+                                    light ? styles.lightError : styles.error,
+                                ]}
+                                role="img"
+                                aria-label={labels.errorIconAriaLabel}
+                            />
+                            <LabelSmall
+                                style={[
+                                    styles.textWordBreak,
+                                    styles.errorMessage,
+                                    light ? styles.lightError : styles.error,
+                                ]}
+                                id={errorId}
```

Comment:
**question:** I wonder if this should be moved to the `View` above. That way, the field could be announced as "some description.... **Error:** the error message" instead of just "some description.... the error message". wdyt?

Reply:
Good question! The `aria-describedby` is using `errorSectionId` instead of `errorId` so that the "Error:" prefix on the icon aria-label is included in the announcement! 

https://github.com/Khan/wonder-blocks/blob/2b809b7d46afbf8bfab02559afb5778ad3723992/packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx#L216-L219

That said, I think having both errorSectionId and errorId can be confusing, and we probably don't need both! I will keep `errorId` and move it to the `View` wrapping the icon and message :)

Reply:
I updated things so that there is only `errorId` on the element wrapping the icon and message! The unit tests were also updated to rely on testIds more to access the error section!